getx_csv_translation_generator/lib/getx_csv_translation_generator.dart

```diff
@@ -42,8 +42,13 @@ class GetXCSVTranslationGenerator
       if (singleQuote) {
         jsonData = formatSingleQuote(jsonData);
       }
+      Map<String, String> keysMap = keys['en'];
```

Comment:
```suggestion
      Map<String, String> keysMap = keys.keys.first;
```
'en' ไม่ได้การันตีว่ามีนะ